src/main/docker/docker-compose.yml

```diff
@@ -135,12 +135,10 @@ services:
     depends_on:
       - rabbitmq
       - mongodb
+    volumes:
+      - "./jenkins/security.groovy:/usr/share/jenkins/ref/init.groovy.d/security.groovy"
     environment:
-      - JENKINS_USERNAME=admin
-      - JENKINS_PASSWORD=admin
-      #     - JENKINS_HOME=                      # JENKINS_HOME: Jenkins home directory. Default: /opt/bitnami/jenkins/jenkins_home
-      #     - DISABLE_JENKINS_INITIALIZATION=yes    # DISABLE_JENKINS_INITIALIZATION: Allows to disable the initial Bitnami configuration for Jenkins. Default: no
-      #     - JAVA_OPTS=                         # JAVA_OPTS: Customize JVM parameters. No defaults.
+      - JAVA_OPTS="-Djenkins.install.runSetupWizard=false"
```

Comment:
Could you document why we turn off the setup wizard?

Reply:
This one is to make sure we skip the part of unlocking and selecting plugins to install that usually pops up when you start jenkins the first time.

Reply:
I've added a comment on this line.